R/compute_age_years.R

```diff
@@ -0,0 +1,67 @@
+#' Compute Age in Years
+#'
+#' Converts a set of age values from the specified time unit to years.
+#'
+#' @param age The ages to convert.
+#'
+#'   A numeric vector is expected.
+#'
+#' @param age_unit Age unit.
+#'
+#'   Either a string containing the time unit of all ages in `age` or a character
+#'   vector containing the time units of each age in `age` is expected. Note that
+#'   permitted values are cases insensitive (e.g. 'YEARS' is treated the same
+#'   as 'years' and 'Years').
+#'
+#'   Permitted Values: 'years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds'.
```

Comment:
Please use backticks and double quote, e.g., `"YEARS"` instead of 'YEARS'.

Reply:
Sorry, I was not clear. I meant to use `` `"YEARS"` ``. Then it is displayed as `"YEARS"`.

Reply:
Got it, thanks for clarifying. It now looks like this:
![image](https://user-images.githubusercontent.com/113363343/234559559-3ccc675f-990b-4208-8309-5037150dc843.png)